src/Form/EmbedButtonForm.php

```diff
@@ -168,6 +168,39 @@ protected function copyFormValuesToEntity(EntityInterface $entity, array $form,
   }
 
   /**
+   * Builds a list of entity type labels suitable for embed button options.
+   *
+   * Configuration entity types without a view builder are filtered out while
+   * all other entity types are kept.
+   *
+   * @return array
+   *   An array of entity type labels, keyed by entity type name.
+   */
+  protected function getFilteredEntityTypes() {
+    $options = array();
+    $definitions = $this->entityManager->getDefinitions();
+
+    foreach ($definitions as $entity_type_id => $definition) {
+      // Don't include configuration entities which do not have a view builder.
+      if ($definition->getGroup() != 'configuration' || $definition->hasViewBuilderClass()) {
```

Comment:
Hrm. @cs-shadow Maybe another way thinking about this is once we have EntityDisplayPluginManager::getDefinitionsForEntityType() (from https://www.drupal.org/node/2349371), we could see which entity types have display plugins, and use that to filter the list, instead of this logic.